hudi-aws/src/main/java/org/apache/hudi/aws/sync/AWSGlueCatalogSyncClient.java

```diff
@@ -467,14 +484,17 @@ public boolean updateTableProperties(String tableName, Map<String, String> table
     }
   }
 
-  private void setComments(List<Column> columns, Map<String, Option<String>> commentsMap) {
-    columns.forEach(column -> {
+  private List<Column> setComments(List<Column> columns, Map<String, Option<String>> commentsMap) {
```

Comment:
The original logic does not take any effect right?

Reply:
That's the point. BTW the test did prove the original method fail. New implem fixes the tests